helm-chart/values.yaml.changelog.md

```diff
@@ -5,6 +5,26 @@ For changes that require manual steps other than changing values, please check o
 Please follow this convention when adding a new row
 * `<type: NEW|EDIT|DELETE> - *<resource name>*: <details>`
 
+----
+## Upgrading to Renku 0.30.0
+* NEW - *ingress.className* is now available to select a specific IngressClass to
```

Comment:
Could you please clarify here if existing deployments which _do not_ need this functionality need to make any changes?

Reply:
Sure thing

Reply:
thanks!!